docs/elements/dynamicscrm/ceql.md

```diff
@@ -1,14 +1,107 @@
 ---
 heading: Microsoft Dynamics CRM
-seo: Query Language | MS Dynamics CRM | Cloud Elements API Docs
+seo: Query Language | Microsoft Dynamics CRM | Cloud Elements API Docs
 title: Querying
 description: Make database queries using Cloud Elements Query Language.
 layout: sidebarelementdoc
 breadcrumbs: /docs/elements.html
-elementId: 190
+elementId: 62
 parent: Back to Element Guides
 order: 45
 sitemap: false
 ---
 
-{% include ceql.md %}
+# Cloud Elements Query Language
+
+The Cloud Elements Query Language (CEQL) is a query language used by Cloud Elements to standardize searching across all of our different elements. Many APIs support some form of searching in their APIs but they’re almost all different, so we have standardized a common way to search across all of our elements. Cloud Elements translates the CEQL to the endpoint’s searching syntax, however at times, CEQL supports more than the endpoint.
+
+## CEQL in {{page.heading}}
+
+When querying in {{page.heading}}, keep the following in mind:
+
+* The {{page.heading}} bulk chunking mechanism allows you to use bulk on large data sets without limitations. If you use an orderBy clause, chunking is disabled. If chunking is enabled, it may take longer to get all the data. Also, be aware that {{page.heading}} has daily upload and download limits.
```

Comment:
I'll update the template to clarify, but this is more of an example of what could be in here.